front/app/components/IdeaCards/IdeasWithFiltersSidebar/ContentRight.tsx

```diff
@@ -17,13 +16,12 @@ import InputFilters from './InputFilters';
 import { gapWidth } from '.';
 
 const Container = styled.div<{
-  filterColumnWidth: number;
   top: number;
   maxHeightOffset: number;
   gapWidth: number;
 }>`
-  flex: 0 0 ${({ filterColumnWidth }) => filterColumnWidth}px;
-  width: ${({ filterColumnWidth }) => filterColumnWidth}px;
+  flex: 0 0 352px;
+  width: 352px;
   display: flex;
```

Comment:
Now that we're using 1 column for the idea card, we can use just a single width for the filter column (so I removed this prop).